test/unit/search/search_collection_test.cpp

```diff
@@ -174,20 +174,21 @@ TYPED_TEST(search_test, multiple_queries)
 
 TYPED_TEST(search_test, parallel_queries)
 {
-    std::vector<std::vector<seqan3::dna4>> const queries{100, {"ACGTACGTACGT"_dna4}};
+    constexpr size_t const num_queries{100u};
```

Comment:
```suggestion
    constexpr size_t num_queries{100u};
```
That is already a constant expression :)